lib/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order
+  attr_accessor :id, :customer_name, :item_id, :date, :items
```

Comment:
I wouldn't necessarily know what the attribute ':items' was if you hadn't explained. Maybe it's worth adding a comment here just to clarify its function.